src/app.md

```diff
@@ -234,6 +235,33 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-file-sizes]: #ex-file-sizes
+<a name="ex-file-sizes"></a>
+## Recursively calculate file sizes at given depth
+
+min_depth & max_depth usage example to calculate total file size at given depth.
```

Comment:
Also please describe in short what the calls to *_depth methods actually do here.